seacrowd/sea_datasets/typhoon_yolanda_tweets/typhoon_yolanda_tweets.py

```diff
@@ -0,0 +1,139 @@
+import os
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_SUPPORTED_TASKS = [Tasks.NAMED_ENTITY_RECOGNITION, Tasks.DEPENDENCY_PARSING]
```

Comment:
_SUPPORTED_TASKS is declared twice, please consolidate

Reply:
deleted this one in the new commit